util/src/main/java/io/zeebe/util/StringUtil.java

```diff
@@ -7,11 +7,28 @@
  */
 package io.zeebe.util;
 
+import static java.util.function.Predicate.not;
+import static java.util.stream.Collectors.toList;
+
 import java.nio.charset.Charset;
 import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.Function;
 
 public final class StringUtil {
 
+  /** Helper functions that removes nulls and empty strings and trims all remaining strings */
+  public static final Function<List<String>, List<String>> LIST_SANITIZER =
```

Comment:
```suggestion
  public static final UnaryOperator<List<String>> LIST_SANITIZER =
```

That said, what is the advantage of declaring it like this versus declaring it as a static method?

Reply:
probably none